src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { PeopleTable } from '../components/PeopleTable/PeopleTable';
+
+export const PeoplePage = () => {
+  const [peoples, setPeople] = useState<Person[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [loader, setLoader] = useState(false);
+
+  useEffect(() => {
+    setLoader(true);
+
+    getPeople()
+      .then((loadPeoples: Person[]) => {
+        setPeople(loadPeoples);
+
+        if (loadPeoples.length > 0) {
+          setErrorMessage('');
+        }
+      })
+      .catch(error => {
+        setErrorMessage('Something went wrong');
+        throw error;
```

Comment:
Throwing an error inside the `.catch()` block will not be caught by any other error handler in this promise chain. Instead of throwing the error, consider logging it or handling it in a way that provides feedback to the user or developers, such as sending it to an error tracking service.